recipe/meta.yaml

```diff
@@ -1,15 +1,16 @@
-{% set version = "0.60" %}
+{% set version = "1.0.0" %}
 
 package:
     name: prompt_toolkit
     version: "{{ version }}"
 
 source:
-    fn: prompt_toolkit-{{ version }}.tar.gz
-    url: https://pypi.python.org/packages/source/p/prompt_toolkit/prompt_toolkit-{{ version }}.tar.gz
-    md5: 516826aff6a09536037b6d02a9a3451d
+    fn: {{ version }}.tar.gz
+    url: https://github.com/jonathanslenders/python-prompt-toolkit/archive/1.0.0.tar.gz
```

Comment:
This could be `https://github.com/jonathanslenders/python-prompt-toolkit/archive/{{ version }}.tar.gz` right?